### PR TITLE
94-use waves svg for dark theme issue is fixed

### LINKED ISSUE #94: Use Waves SVG for Dark Theme
#### Describe the bug
The create-account success page in the custom example is using an image that only looks good in the light theme.  React workflow example is using a waves svg that looks good in both light and dark, use that one here. 

#### What is the expected behavior?
The picture should look good on dark and light themes. 

#### What are the steps to reproduce?
1.  Clone the repo (`git clone https://github.com/brightlayer-ui/angular-workflows.git`).
2. CD into the project (`cd angular-workflows`)
3. Run the example project (`cd login-workflow && yarn start:example`)
4. Navigate to http://localhost:4200/auth/register/create-account
5. Inspect the page & update the `<body>` class from `blui-blue` to `blui-blue-dark`.  This will set the dark theme manually.
6. Fill in the required information to create an account.
7. Observe the final page; it should match the image below. 

#### Screenshots or links to minimum reproduction example
image

### PR DESCRIPTION
**Pull request by raghavenc5**
Fixes #94 

#### Changes proposed in this Pull Request:
-  Changed the SVG transparent background image to resolve this issue
- 
- 

#### Screenshots / Screen Recording (if applicable)
-
image

#### To Test:
- yarn test

#### Any specific feedback you are looking for?
-

### CODE REVIEW

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v4.0.1 (Unreleased)
+
+### Fixed
+
+-   Fixed Use waves SVG image for dark theme issue ([#94](https://github.com/etn-ccis/blui-angular-workflows/issues/94)).
```
Comment (surajeaton) [inline login-workflow/CHANGELOG.md:7]: This should be something meaningful like:

### Changed

-   Updated wave image to registration success screen ([#94](https://github.com/etn-ccis/blui-angular-workflows/issues/94)).

Comment (joebochill) [reply login-workflow/CHANGELOG.md:7]: This actually shouldn't be mentioned in the changelog at all right? This is just a change in the demo project not a change in the package...

**Review by huayunh — COMMENTED** [review]
- Agree with Joe. This does not need a changelog item or version change in package.json 
- Remember to remove unused assets, like `/login-workflow/example/src/assets/images/waves.svg`
- code change otherwise looks good.

**Comment by huayunh** [comment]
see comments above about changelog.

File: login-workflow/CHANGELOG.md (hunk 1 above)
Comment (raghavenc5) [reply login-workflow/CHANGELOG.md]: Revoked the changes in the changelog and package file

**Comment by raghavenc5** [comment]
> see comments above about changelog.

Revoked the changes in the changelog and package file

**Pull request merged**